core/src/main/resources/jenkins/management/AdministrativeMonitorsDecorator/resources.css

```diff
@@ -100,37 +123,46 @@
     font-weight: 600;
     font-weight: var(--link-font-weight);
 }
-#visible-am-container #visible-am-list a:visited {
+.am-container .am-list a:visited {
     color: #5c3566;
     color: var(--link-color);
 }
-#visible-am-container #visible-am-list a:hover,
-#visible-am-container #visible-am-list a:focus,
-#visible-am-container #visible-am-list a:active {
+.am-container .am-list a:hover,
+.am-container .am-list a:focus,
+.am-container .am-list a:active {
     color: #5c3566;
     color: var(--link-color);
     background-color: transparent;
     text-decoration: underline;
     text-decoration: var(--link-text-decoration--hover);
 }
 
-#visible-am-container #visible-am-list .alert-success a {
+.am-container .am-list .alert-success a {
     color: #155724;
     color: var(--alert-success-text-color);
 }
 
 
-#visible-am-container #visible-am-list .alert-info a {
+.am-container .am-list .alert-info a {
     color: #31708f;
     color: var(--alert-info-text-color);
 }
 
-#visible-am-container #visible-am-list .alert-warning a {
+.am-container .am-list .alert-warning a {
     color: #8a6d3b;
     color: var(--alert-warning-text-color);
 }
 
-#visible-am-container #visible-am-list .alert-danger a {
+.am-container .am-list .alert-danger a {
     color: #a94442;
     color: var(--alert-danger-text-color);
 }
+
+@media screen and (max-width: 768px) {
```

Comment:
```suggestion
@media screen and (max-width: 576px) {
    /* Hide non-security monitors on mobile view to avoid messing up the heading */
    #visible-am-container {
       display: none;
    }
}

@media screen and (max-width: 768px) {
```